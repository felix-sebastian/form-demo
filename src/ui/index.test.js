import React from "react";
import Space from "./space";
import renderer from "react-test-renderer";

it("defaults to horizontal space", () => {
  const result = renderer.create(<Space />).toJSON();
  expect(result).toMatchSnapshot();
});

it("can render vertical space", () => {
  const result = renderer.create(<Space vertical />).toJSON();
  expect(result).toMatchSnapshot();
});
