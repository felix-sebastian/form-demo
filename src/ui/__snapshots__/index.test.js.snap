// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render vertical space 1`] = `
<div
  className="sc-AykKD kYfJla"
  size={1}
/>
`;

exports[`defaults to horizontal space 1`] = `
<div
  className="sc-AykKC jujAmj"
  size={1}
/>
`;
